backend/v2/epochs/dependencies.py

```diff
@@ -61,7 +56,48 @@ def get_epochs_subgraph(
     return EpochsSubgraph(settings.subgraph_endpoint)
 
 
+async def get_current_epoch(epochs_contracts: GetEpochsContracts) -> int:
+    return await epochs_contracts.get_current_epoch()
+
+
+async def get_indexed_epoch(epochs_subgraph: GetEpochsSubgraph) -> int:
+    sg_epochs = await epochs_subgraph.get_epochs()
+    sg_epochs_sorted = sorted(sg_epochs, key=lambda d: d.epoch_num)
+
+    return sg_epochs_sorted[-1].epoch_num
+
+
+async def get_rewards_rate(epoch_number: int) -> float:
+    logging.debug(f"Getting rewards rate for epoch {epoch_number}")
+    return ESTIMATED_STAKING_REWARDS_RATE
```

Comment:
It's also just a passthru call on a variable, Maybe reduce this to just reference static ESTIMATED_STAKING_REWARDS_RATE in the router response directly? Saving some space and unnecesasary declaration

Reply:
yeah but i put in a separate place to address our need of assessing ESTIMATEAD_STAKING_REWARDS_RATE dynamically, not statically (https://linear.app/golemfoundation/issue/OCT-902/api-call-to-get-validators-api);

looking at such isolated thing always make me think it's open to be changed in the future but I can move it around if you stick to it